### PR TITLE
feat(TimeSeriesCard/BarChartCard): add zoomBar functionality

### PR DESCRIPTION
**Pull request by sstone2423**
Closes https://github.com/carbon-design-system/carbon-addons-iot-react/issues/1456

**Summary**

- Adds alpha zoomBar functionality to TimeSeriesCard and BarChartCard using the new `zoomBar` prop
- Restricts the zoomBar to cards of size MEDIUMWIDE, LARGE, LARGEWIDE as the other sizes are too small to render both the chart and the zoomBar
- Note: This feature is experimental in carbon charts
```
zoomBar: {
  axes: string,   // defaults to top as thats the only axis supported at the moment
  enabled: boolean,
  initialZoomDomain: array<string|number>    // same as the current `domainRange` prop
}
```

**Change List (commits, features, bugs, etc)**

- Added 2 stories to show the zoomBar feature
- Updated to latest carbon charts to pick up this feature

**Acceptance Test (how to verify the PR)**

- Go to BarChartCard and TimeSeriesCard `with zoomBar` stories and test out the zoom bar

### CODE REVIEW

File: src/components/TimeSeriesCard/TimeSeriesCard.jsx [hunk 1]
```diff
@@ -68,6 +73,17 @@ const TimeSeriesCardPropTypes = {
     ),
     /** optional units to put in the legend */
     unit: PropTypes.string,
+    /** Optionally addes a zoom bar to the chart */
+    zoomBar: PropTypes.shape({
+      /** Determines which axis to put the zoomBar */
+      axes: PropTypes.string,
```
Comment (tay1orjones) [inline src/components/TimeSeriesCard/TimeSeriesCard.jsx:79]: I'm not familiar with the prop model that much. Is there a specific set of axes? What format should the string be in? Wondering if we should make it an enum for clarity
```
axes: PropTypes.oneOf(['x-axis', 'y-axis']),
```
or
```
axes: PropTypes.oneOf(['top', 'bottom', 'left']),
```

Comment (tay1orjones) [reply src/components/TimeSeriesCard/TimeSeriesCard.jsx:79]: I think we also would need to update this in `CardPropTypes`

Comment (sstone2423) [reply src/components/TimeSeriesCard/TimeSeriesCard.jsx:79]: Oh good call. I'll update it to be `axes: PropTypes.oneOf(['top', 'bottom', 'left']),` , where I could actually enforce the `'top'` until it the other 3 are supported

**Pull request merged**